packages/web3torrent/src/App.tsx

```diff
@@ -9,12 +9,21 @@ import File from './pages/file/File';
 import Upload from './pages/upload/Upload';
 import {RoutePath} from './routes';
 import {requiredNetwork} from './constants';
+import {Budgets} from './pages/budgets/Budgets';
+import {InitializationContext} from './contexts/initialization-context';
 
 const App: React.FC = () => {
   const [currentNetwork, setCurrentNetwork] = useState(
     window.ethereum ? Number(window.ethereum.networkVersion) : undefined
   );
 
+  const {initialize, isInitialized} = useContext(InitializationContext);
```

Comment:
This isn't strictly necessary since `initialize` will be called whenever another context is used, but it's nice to start loading the `iFrame` as soon as we hit the page.